webapp/channels/src/components/integrations/abstract_command.tsx

```diff
@@ -113,11 +97,19 @@ export default class AbstractCommand extends React.PureComponent {
             icon_url: this.state.iconUrl,
             auto_complete: this.state.autocomplete,
             team_id: this.props.team.id,
+            auto_complete_desc: '',
+            auto_complete_hint: '',
+            token: '',
+            create_at: 0,
+            update_at: 0,
+            delete_at: 0,
+            id: '',
+            creator_id: '',
```

Comment:
why did we had to reset the values here?

Reply:
We're initializing the object here so that we don't get typescript error like this
<img width="586" alt="Screenshot 2023-05-08 at 5 49 22 PM" src="https://user-images.githubusercontent.com/43727167/236822062-9a068530-421b-4060-aae8-44175d257c1a.png">